monitoring/grafana/dashboards/tests/features/osd-device-details.feature

```diff
@@ -0,0 +1,22 @@
+Feature: OSD device details
+
+Scenario: "Test $osd onode Hit, Miss Requests"
+  Given the following series:
+    | metrics | values |
+    | ceph_bluestore_bluestore_onode_hits{ceph_daemon="osd.0",instance="ceph:9283",job="ceph"} | 5255 |
```

Comment:
`ceph_bluestore_bluestore_onode_hits` ok, I think I now understand that the clean-up in the RADOS perf-counters was to remove this redundancy, @ljflores ? :S

Reply:
@epuertat yes, and I believe #44334 mainly affects the output of `ceph tell osd.* perf dump` by changing the name of the perf counter and without modifying the perf counter "key".

Reply:
@ljflores I'm afraid that the name (label( of the perf counter is actually the key (I don't see any other string literal over there, and AFAIK C++ doesn't support reflection... yet). Anyway, as I commented in that PR, given this 2 are the only metrics left with this redundant format, let's go for it.